platform_api/orchestrator/base_postgres_storage.py

```diff
@@ -21,7 +23,7 @@ async def _execute(
         if conn:
             await conn.execute(query)
             return
-        async with self._engine.connect() as conn:
+        async with self._transaction() as conn:
```

Comment:
Do we need a transaction here? Would not `_safe_connect(self._engine.connect())` be enough?

Reply:
Yes, thanks, you are right, for some reason I replaced all `connect()` with `begin()` when was moving out wrapper to a separate method.